scripts/change_url

```diff
@@ -8,6 +8,7 @@
 
 source _common.sh
 source /usr/share/yunohost/helpers
+source _sed
```

Comment:
... and if we depend on 2.7.7, we could maybe have a much easier script... ? 😉

Reply:
Yes we could, but as a security fix, it would be better to merge that before. Then work on a refactoring of the change_url script.

Reply:
Agreed!